android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -410,6 +459,24 @@ private void registerDataChannelObserver(int dcId, DataChannel dataChannel) {
             new DataChannelObserver(webRTCModule, id, dcId, dataChannel));
     }
 
+    /**
+     * Removes given local <tt>MediaStream</tt> from the underlying
+     * <tt>PeerConnection</tt>.
+     * @param localMediaStream a local <tt>MediaStream</tt>
+     *
+     * @return <tt>true</tt> if the underlying <tt>PeerConnection</tt> exists
+     * and the given <tt>MediaStream</tt> was in the local streams collection.
+     */
+    boolean removeStream(MediaStream localMediaStream) {
+        if (this.localStreams.remove(localMediaStream.label()) != null
```

Comment:
* We don't need to check whether `localStream` contains `localMediaStream`. First remove from `peerConnection` and, once it doesn't throw, remove from localStreams. The `remove` will give you what it has removed so the return value should be based on the return value of `remove`.

* Please call it `localStream` not `localMediaStream` like `localStreams`, `remoteStreams`, etc. 

* Please drop the use of `this` unless it's mandatory.